app/assets/stylesheets/components/common/_collapsable-mobile.scss

```diff
@@ -0,0 +1,26 @@
+.collapsable-mobile {
+
+  .collapsable__trigger-icon {
+    float: right;
+    margin: 10px 20px 0 5px;
+
+    @include respond-to($mq-m) {
+      display: none;
+    }
+  }
+
+  button {
```

Comment:
Unfortunately, in this situation we can't use pure BEM, the parent Collapsable Component injects the button.

the ```. collapsable__trigger-icon``` is an element of the nested (Parent) component and I can't flatten that out of the nested class otherwise it'll apply to and therefore break all the other Collapsable components.